build.sbt

```diff
@@ -123,8 +124,8 @@ lazy val scala = Project(id = "content-atom-model", base = file("scala"))
       (scroogeUnpackDeps in Compile).value.flatMap { dir => (dir ** "*.thrift").get }
     },
     libraryDependencies ++= Seq(
-      "org.apache.thrift" % "libthrift" % "0.9.2",
-      "com.twitter" %% "scrooge-core" % "4.5.0"
+      "org.apache.thrift" % "libthrift" % "0.9.3",
+      "com.twitter" %% "scrooge-core" % "4.12.0"
```

Comment:
https://github.com/twitter/scrooge/blob/develop/scrooge-core/src/main/scala/BUILD suggests that it's been built against the 0.5.0 release for a long time now. Difficult to tell if that 's a forked version or not. If this worked with 0.9.2, I suspect it would work with 0.9.3, but I'm happy to drop that back down if we're nervous about it. 4.12.0 is the first 2.12 published version of scrooge-core.